src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -120,10 +120,14 @@ public function title($message)
      */
     public function section($message)
     {
+        // don't take into account the style tags to compute the message length
+        // otherwise, the underline displayed after the section title is too long
+        $messageLength = strlen(preg_replace('/<.*>/U', '', $message));
```

Comment:
Indeed, you can actually use [`Helper::strlenWithoutDecoration`](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Console/Helper/Helper.php#L109-L120)